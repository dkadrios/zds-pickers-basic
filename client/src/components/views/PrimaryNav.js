import React from 'react'
import UserInfo from '../user/UserInfo'
import MainMenu from '../MainMenu'
import Logo from '../../images/shifter.svg'
import EditEntryDlg from '../entries/EditEntryDlg'
import ErrorDialog from '../ErrorDialog'
import { mainContent, topBar } from '../../styles/primaryNav.scss'
import Documentation from '../Documentation'
import ShiftGroups from './ShiftGroups'
import InputControls from './InputControls'

const PrimaryNav = () => (
  <div>
    <div className={topBar}>
      <Logo />
      <section>
        <UserInfo />
        <Documentation />
        <MainMenu />
      </section>
    </div>

    <section className={mainContent}>
      <InputControls />
      <ShiftGroups />
      <EditEntryDlg />
      <ErrorDialog />
    </section>
  </div>
)

export default PrimaryNav
