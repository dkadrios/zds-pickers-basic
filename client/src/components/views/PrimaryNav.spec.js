import PrimaryNav from './PrimaryNav'

describe('PrimaryNav tests', () => {
  beforeEach(jest.clearAllMocks)

  it('renders correctly', () => {
    shallowExpect(PrimaryNav)({}).toMatchSnapshot()
  })
})
