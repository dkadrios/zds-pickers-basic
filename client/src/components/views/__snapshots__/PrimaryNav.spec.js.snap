// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryNav tests renders correctly 1`] = `
<div>
  <div>
    <test-file-stub />
    <section>
      <Connect(UserInfo) />
      <Documentation />
      <Connect(MainMenu) />
    </section>
  </div>
  <section>
    <Connect(InputControls) />
    <Connect(ShiftGroups) />
    <Connect(EditEntryDlg) />
    <Connect(ErrorDialog) />
  </section>
</div>
`;
